shared/local-debug-live.js

```diff
@@ -1,4 +1,15 @@
-export const dumbFilter = ''
+// @flow
+let envJson = {}
+if (process.env.KEYBASE_LOCAL_DEBUG_JSON) {
+  try {
+    envJson = JSON.parse(process.env.KEYBASE_LOCAL_DEBUG_JSON)
+  } catch (e) {
+    envJson = {}
+    console.warn('Invalid KEYBASE_LOCAL_DEBUG_JSON:', e)
+  }
+}
+
+export const dumbFilter = envJson.dumbFilter || ''
```

Comment:
I'm not exactly clear what you mean here. The envJson.dumbFilter || '' was meant to ensure the string had a value always. if you want to hot load and edit this file to change the filter wouldn't you just do 
```
dumbFilter = 'foo'
```
?